generator/generate.js

```diff
@@ -294,7 +295,7 @@ ${generateFragments(name, primitiveFields, nonPrimitiveFields)}
     let modelProperties = ""
     if (type.fields) {
       modelProperties = type.fields
-        .filter(field => field.args.length === 0)
```

Comment:
I'm curious why this line was here in the first place (@mweststrate) but if things work fine without it I'm all for just removing it entirely vs. adding an `allowFieldArgs` option.

Reply:
hmm yeah, I thought maybe orginally it was meant as `field.args.length !== 0`  but does not make sense neither down the line. I agree @chrisdrackett just drop only that line (would as well reduce all the unreleated formating change)  :trollface: